### ISSUE TITLE
a11y: Search simple component svg no alt text and minimum contrast issue

### ISSUE THREAD

**Issue by shjunmiao** [carbon-design-system/carbon-components-react#334]
We are using the Search small component, and when we run DAP showing below
violations.  See screenshot.

Could you plz help address those? 
<img width="1315" alt="screen shot 2017-11-13 at 15 32 58" src="https://user-images.githubusercontent.com/31211767/32734229-4d79cf7e-c889-11e7-9e41-5c18b2d2d7fa.png">

**Comment by tw15egan** [comment]
Hi, since SVG's don't have the alt attribute, it uses `title` and `aria-label` instead to make sure it's accessible - therefore this component is not violating any rules. Also since the input uses opacity to calculate its background color, the svg throws an unnecessary color contrast warning. Feel free to reopen this if you feel this is closed in error.

**Comment by tw15egan** [comment]
<img width="330" alt="screen shot 2017-11-13 at 10 12 15 am" src="https://user-images.githubusercontent.com/11928039/32735784-47c9f776-c85b-11e7-81ef-1c90abc6ce52.png">
